lib/experimental.ts

```diff
@@ -4,15 +4,41 @@ import * as path from "path";
 
 import { pullIfNotLocal } from "./analyzer/image-inspector";
 import { Docker } from "./docker";
+import { getDockerArchivePath, getImageTransport } from "./image-transport";
 import * as staticModule from "./static";
-import { ImageType, PluginResponse } from "./types";
+import {
+  ImageTransport,
+  ImageType,
+  PluginResponse,
+  StaticAnalysisOptions,
+} from "./types";
 
 export async function experimentalAnalysis(
   targetImage: string,
   options: any,
 ): Promise<PluginResponse> {
   // assume Distroless scanning
-  return distroless(targetImage, options);
+  const imageTransport = getImageTransport(targetImage);
+  switch (imageTransport) {
+    case ImageTransport.DockerArchive:
+      // TODO: maybe all the steps in this case can be wrapped in a function
```

Comment:
yup, we should have a function that gets a tar file path on the disk and returns the scan. that way the distroless flow (which should be called in a different name I guess now, something like containerRegsitryAnalysis) can call it after it calls docker save